src/server/auth/server/admin_test.go

```diff
@@ -1507,3 +1508,187 @@ func TestDeleteRCInStandby(t *testing.T) {
 		return err
 	})
 }
+
+// TestNoOutputRepoDoesntCrashPPSMaster creates a pipeline, then deletes its
+// output repo while it's running (failing the pipeline and preventing the PPS
+// master from finishing the pipeline's output commit) and makes sure new
+// pipelines can be created (i.e. that the PPS master doesn't crashloop due to
+// the missing output repo)
+//
+// Note: arguably deleting the output repo of a pipeline, even one that's
+// stopped, should be prevented. However, the way that PPS currently uses PFS
+// (stopping a pipeline = removing output branch subvenance) we have no way to
+// prevent that. Thus, this test at least makes sure that doing such a thing
+// doesn't break the PPS master.
+//
+// Note: This test actually doesn't use the admin client or admin privileges
+// anywhere. However, it restarts pachd, so it shouldn't be run in parallel with
+// any other test
```

Comment:
I don't think tests in `src/server/pachyderm_test.go` run in parallel either, because they all delete everything as the first step in the test, so it's hard to see how they would pass if they did.

Reply:
Did you figure out if this was actually true and if so can this test be put with the other pps tests rather than the auth tests.

Reply:
Sorry I had a response but I accidentally did it as a review instead of a one-off comment so it was stuck in pending. Basically I'd be happy to put this in `auth_test.go` rather than `admin_test.go`, but it is intended to run with auth on